src/app/projects/add-edit-project/add-edit-project.component.html

```diff
@@ -75,6 +75,12 @@ <h2>Basic Information</h2>
               </div>
           </div>
         </div>
+        <div class="label-pair med">
+          <div class="form-group">
+            <label for="logos">Project Logos </label>
```

Comment:
Did you mean to add the space, behind Logos, if your trying to add space, maybe styling might be a better option... your call...